src/compiler/checker.ts

```diff
@@ -15358,16 +15321,27 @@ namespace ts {
             return getUnionType(map(instantiatedSignatures, getReturnTypeOfSignature), UnionReduction.Subtype);
         }
 
+        function getJsxNamespaceForLocation(location: Node) {

```

Comment:
Super nit, the naming of getJsxNamespace**ForLocation** should probably match getJsxStatelessElementType**At** / getJsxElementClassType**At**